lllc/main.cpp

```diff
@@ -52,10 +53,20 @@ void version()
 	exit(0);
 }
 
+void setEnv() {
+	std::setlocale(LC_ALL, "C");
+#if !defined(WIN32) && !defined(MAC_OSX) && !defined(__FreeBSD__) && !defined(__OpenBSD__)
```

Comment:
To simplify things, I guess it is fine to just stay at the `C`-locale. I hope we do not use any locale-specific code in the compiler itself (i.e. the generated code should not depend on that). The only other locale-specific thing I can think of are low-level error messages, and I think it is not too bad if they are not localized.